kinto/group_vars/kinto_web_stage.yml

```diff
@@ -0,0 +1,61 @@
+apache_remove_default_vhost: true
+apache_vhosts_filename: "ssl.conf"
+
+apache_global_vhost_settings: |
+    LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+    LogFormat "%h %l %u %t \"%r\" %>s %b" common
+    LogFormat "%v: %h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" vcombined
+    CustomLog "logs/access_log" vcombined
+    
+    ServerLimit 32
+    StartServers 8
+    MaxRequestWorkers 1024
+    MinSpareThreads 25
+    MaxSpareThreads 75
+    ThreadsPerChild 32
+
+    AddOutputFilterByType DEFLATE text/plain
+    AddOutputFilterByType DEFLATE text/html
+    AddOutputFilterByType DEFLATE text/xml
+    AddOutputFilterByType DEFLATE text/css
+    AddOutputFilterByType DEFLATE application/xml
+    AddOutputFilterByType DEFLATE application/xhtml+xml
+    AddOutputFilterByType DEFLATE application/rss+xml
+    AddOutputFilterByType DEFLATE application/javascript
+    AddOutputFilterByType DEFLATE application/x-javascript
+
+    LoadModule ssl_module modules/mod_ssl.so
+    Listen 443
+    SSLPassPhraseDialog  builtin
+    SSLSessionCache         shmcb:/var/cache/mod_ssl/scache(512000)
+    SSLSessionCacheTimeout  300
+    Mutex default
+    SSLRandomSeed startup file:/dev/urandom  256
+    SSLRandomSeed connect builtin
+    SSLCryptoDevice builtin
+apache_vhosts_template: "../../vars/vhost_template.j2"
+apache_certificate_key_file: "/etc/httpd/conf/privkey.pem"
```

Comment:
The ELB should be handling the SSL certs, not Apache, so none of the SSL stuff is needed here.

Reply:
copied that from the old kinto config and forgot to remove it.  it'll go away soon.

Reply:
and it's gone.